packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/Evm-artifacts/L1Bridge.abi

```diff
@@ -0,0 +1,485 @@
+[
```

Comment:
Let's try to import the contracts we need (source or artifact) using a package manager.

Please figure out how to rebase these artifacts out of this PR and our git history.

Reply:
They should probably be included in `@chainlink-dev/starkgate-contracts` @sdrug